pom.xml

```diff
@@ -47,7 +47,7 @@ under the License.
     </modules>
 
     <properties>
-        <flink.version>1.17.0</flink.version>
+        <flink.version>1.18.0</flink.version>
```

Comment:
This isn't correct, this should be set to the lowest Flink version that's supported

Reply:
reverted